packages/resource-deployment/scripts/create-cosmos-db.sh

```diff
@@ -0,0 +1,143 @@
+            --database-name "$dbName" \
+            --name "$collectionName" \
+            --resource-group "$resourceGroupName" \
+            --max-throughput "$throughput" 1>/dev/null
+        echo "Updating ttl for collection '$collectionName'"
+        az cosmosdb sql container update \
+            --account-name "$cosmosAccountName" \
+            --database-name "$dbName" \
+            --name "$collectionName" \
+            --resource-group "$resourceGroupName" \
+            --ttl "$ttl" 1>/dev/null
+
+        echo "Successfully updated collection '$collectionName'"
+    else
+        echo "Collection '$collectionName' does not exist"
+
+        echo "Creating autoscale throughput collection '$collectionName'"
+        az cosmosdb sql container create \
+            --account-name "$cosmosAccountName" \
+            --database-name "$dbName" \
+            --name "$collectionName" \
+            --resource-group "$resourceGroupName" \
+            --partition-key-path "/partitionKey" \
+            --max-throughput "$throughput" \
+            --ttl "$ttl" 1>/dev/null
+
+        echo "Successfully created collection '$collectionName'"
+    fi
+}
+
+createCosmosDatabase() {
+    local dbName=$1
+
+    echo "Checking if database '$dbName' exists in Cosmos account '$cosmosAccountName' in resource group '$resourceGroupName'"
+
+    if az cosmosdb sql database show --name "$dbName" --account-name "$cosmosAccountName" --resource-group "$resourceGroupName" --query "id" 2>/dev/null; then
+        echo "Database '$dbName' already exists"
+    else
+        echo "Creating Cosmos DB '$dbName'"
+        az cosmosdb sql database create \
+            --name "$dbName" \
+            --account-name "$cosmosAccountName" \
+            --resource-group "$resourceGroupName" 1>/dev/null
+        echo "Successfully created Cosmos DB '$dbName'"
+    fi
+}
+
+function setupCosmos() {
+    createCosmosAccount
+
+    local WebInsightsDbName="WebInsights"
+
+    local cosmosSetupProcesses=(
+        "createCosmosDatabase \"$WebInsightsDbName\""
+    )
+    echo "Creating Cosmos databases in parallel"
+    runCommandsWithoutSecretsInParallel cosmosSetupProcesses
+
+    # Increase autoscale maximum throughput for below collection only in case of prod
+    # Refer to https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live for item TTL scenarios
+    if [ $environment = "prod" ] || [ $environment = "ppe" ]; then
+        cosmosSetupProcesses=(
+            "createCosmosCollection \"websiteData\"  \"$WebInsightsDbName\" \"-1\" \"40000\""      # never expire
+            "createCosmosCollection \"scanMetadata\" \"$WebInsightsDbName\" \"7776000\" \"40000\"" # 90 days
+        )
+    else
+        cosmosSetupProcesses=(
+            "createCosmosCollection \"websiteData\"  \"$WebInsightsDbName\" \"-1\" \"4000\""      # never expire
+            "createCosmosCollection \"scanMetadata\" \"$WebInsightsDbName\" \"7776000\" \"4000\"" # 90 days
```

Comment:
Do we plan to delete metadata after 90 days?

Reply:
Changed to infinite ttl for now